deploy/charts/firefly/values.yaml

```diff
@@ -0,0 +1,237 @@
+# Templates the firefly.core configuration file of FireFly, and in some cases configures which ports are exposed.
+config:
+  # Enables the FireFly debug port on 6060 and `DEBUG` level logs
+  debugEnabled: false
+
+  # Enables the Admin API port for dynamic configuration
+  adminEnabled: true
+
+  # Puts a fresh FireFly node into the preinit state, allowing an operator to then setup smart contracts, apply database migrations, etc. before re-configuring the node to proceed.
+  # It is _not_ recommended to configure FireFly nodes in a preinit state for non-development scenarios.
+  preInit: false
+
+  # The name of the organization the FireFly node belongs to
+  organizationName: ""
+
+  # The blockchain identity of the organization e.g. the Ethereum account address
+  organizationIdentity: ""
+
+  # The Ethereum address of the pre-deployed FireFly smart contract
+  fireflyContractAddress: "/instances/contractAddress"
+
+  # The Postgres URL (including credentials and DB) for FireFly to for the database plugin
+  postgresUrl: ""
+
+  # Whether or not to apply schema migrations automatically on startup, not recommended for production
+  postgresAutomigrate: false
+
+  # The URL of the HTTPS DataExchange for the node to use for the dataexchange plugin i.e. private messaging, only needed if `dataexchange.enabled` is set to false
+  dataexchangeUrl: ""
```

Comment:
I think I understand how this is technically a property of the core config (since this gets exposed from the node), but I wonder if this would made more sense as `dataexchange.url`?

Reply:
See https://github.com/hyperledger/firefly/pull/114/files#discussion_r718603904https://github.com/hyperledger/firefly/pull/114/files#discussion_r718603904